src/SDKs/ContainerInstance/Management.ContainerInstance/Generated/SdkInfo_ContainerInstanceManagementClient.cs

```diff
@@ -1,22 +0,0 @@
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-
-internal static partial class SdkInfo
-{
-    public static IEnumerable<Tuple<string, string, string>> ApiInfo_ContainerInstanceManagementClient
-    {
```

Comment:
Why did this get deleted? How was the SDK generated? Was the [generate.ps1](https://github.com/Azure/azure-sdk-for-net/blob/151b8c83af8a005d46beec5e6f6fade8eca4dd49/src/SDKs/ContainerInstance/Management.ContainerInstance/generate.ps1) used?